### ISSUE TITLE
Unum doens't have method for checking if any dirty properties present

### ISSUE THREAD

**Issue by asgaines** [SocialEngine/Unum#1]
Possible implementation:

``` php
public function isClean()
{
    return empty($this->propertyDirty);
}
```

With test:

``` php
public function test_is_clean()
{
    $entity = new TestEntity(['testProp' => 'test1', 'test_prop' => 'test2']);

    $this->assertTrue($entity->isClean());
    $entity->testProp = 'testInfinity';

    $this->assertFalse($entity->isClean());
}
```

**Comment by Danno040** [comment]
If you have an implementation and a test, why is this not a PR?

**Comment by Danno040** [comment]
Resolved by #3
